frontend/components/ListItem.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+import React, { Component } from 'react'
+import { StyleSheet, View, TouchableOpacity, TouchableNativeFeedback } from 'react-native';
+import { Card, Text } from 'react-native-elements';
+import Icons from '@expo/vector-icons/FontAwesome';
+
+/**
+ * ListItem describes a clickable list element that is displayed on the homepage
+ */
+export default class ListItem extends Component {
+
+    render() {
+        return (
+            <TouchableNativeFeedback>
+                <Card containerStyle={styles.card}>
+                    <View style={{ flex: 1, flexDirection: 'row', alignItems: 'center' }}>
```

Comment:
The style for this view doesn't use anything dynamic so it could be moved to stylesheet to improve readibility

Reply:
dang you got me